haystack/components/converters/docx.py

```diff
@@ -0,0 +1,91 @@
+import io
+import logging
+from pathlib import Path
+from typing import Any, Dict, List, Optional, Union
+
+from haystack import Document, component
+from haystack.components.converters.utils import get_bytestream_from_source, normalize_metadata
+from haystack.dataclasses import ByteStream
+from haystack.lazy_imports import LazyImport
+
+logger = logging.getLogger(__name__)
+
+with LazyImport("Run 'pip install docx'") as docx_import:
```

Comment:
```suggestion
with LazyImport("Run 'pip install python-docx'") as docx_import:
```
Update library install to the maintained library.